ta/os_test/sub.mk

```diff
@@ -1,3 +1,7 @@
+ifeq ($(CFG_TA_FLOAT_SUPPORT),y)
+cppflags$(sm) += -DCFG_TA_FLOAT_SUPPORT=1
```

Comment:
`cppflags-$(CFG_TA_FLOAT_SUPPORT) += -DCFG_TA_FLOAT_SUPPORT=1` should be sufficient.